### DISCUSSION TITLE
제페토 카메라 컨트롤

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#54] [5. Q&A]
제페토 캐릭터를 생성하면 ZepetoPlayers오브젝트가 가지고 있는 camera프리펩을 자동으로 붙여주는데
제페토 카메라 스크립트가 자동으로 따라 붙어서 카메라를 임의로 변경할 수 없습니다.
카메라를 따로 제작해서 플레이어에 붙이고 제페토 카메라 스크립트가 붙지 않게 하려면 어떻게 해야 하나요?

**Comment by x4cf** [comment]
카메라가 생성되자마자 꺼버리고 준비된 카메라를 가져오는건 어떨까요?

**Comment by moondory77-naverz** [comment]
안녕하세요. 
캐릭터 컨트롤러 모듈을 통해, LocalPlayer를 생성하고 사용할 경우
ZepetoPlayers 컴포넌트에 Prefab으로 설정된, ZepetoCamera를 생성하여 사용하게 됩니다.

만약 ZepetoCamera를 사용하지 않고 임의의 카메라를 사용하실 경우,       
ZepetoPlayers.instance.LocalPlayer.zepetoCamera.gameObject.SetActive(false);
와 같은 방식으로 접근한 다음  disable 처리 해주시면 될 듯 합니다.

또 다른 문의가 있으실 경우, 언제든 말씀 부탁드립니다.
감사합니다 :)
